### ISSUE TITLE
Suggested revision for vaccinated/booster multiplier for omicron vs delta

### ISSUE THREAD

**Issue by ppanigr** [microCOVID/microCOVID#1294]
Thanks all for your efforts on a project that grows increasingly complex.  There is data on VE on unvaccinated vs vaccinated vs boosted that I believe warrants an update. 

The current multipliers assume 3x mRNA vaccines are 75% effective. I was unable to locate the Ferguson et al reference cited, but in case that was based on neutralizing antibody titers, it's important to note that % reduction in antibody titers =/= % reduction in vaccine efficacy.

Here are two references with thousands of data points on secondary attack rates of omicron and delta with varying levels of vaccination:

Ontario study
https://www.medrxiv.org/content/10.1101/2021.12.30.21268565v1.full-text
Table 2 shows: 3rd pfizer results in 34% VE against omicron and 93% VE against delta; 3rd moderna results in 59% VE against omicron and 93% VE against delta. First two doses at 6% VE and under against omicron as time has lapsed since vaccination, and 84%-->71% VE against delta as time has lapsed since vaccination.
![image](https://user-images.githubusercontent.com/97255430/148444695-edbe4d86-3f78-40c9-9544-e2506d73bf91.png)

Denmark study
https://www.medrxiv.org/content/10.1101/2021.12.27.21268278v1.full#T3
Table 2 shows: "Fully vaccinated" had almost no effect against omicron (1-1/1.04=.04), modest effect against delta (1-1/2.31=.57) with unknown time from vaccination (compared to "unvaccinated"); "Booster-vaccinated" had modest effect against omicron (1-.54/1.04=0.48), high effect against delta (1-.38/2.31=.84).
![image](https://user-images.githubusercontent.com/97255430/148444746-7cc75e33-3115-4baa-8f06-4fbdddfa48bd.png)

**Comment by beshaya** [comment]
Thanks for the studies, definitely stuff for me to chew on. The paper we cited was https://www.imperial.ac.uk/media/imperial-college/medicine/mrc-gida/2021-12-16-COVID19-Report-49.pdf.

The Denmark study blends all "boosted" vaccines, which includes Johnson&Johnson, which has a VE of approximately 0 according to J&J's own study in South Africa https://www.medrxiv.org/content/10.1101/2021.12.28.21268436v1.full.pdf. This would muddy their data, and I don't see a breakdown per-vaccine.

The Ontario study seems mostly legit, although they don't compensate for differences in behavior, so a possible reason for the discrepancy is that vaccinated people are out doing riskier things because they think they're immune. I went and had a look at breakthrough infection rates from a few regions that produce them:
* 68% naive VE in Santa Clara https://covid19.sccgov.org/dashboard-case-rates-vaccination-status
* 78% apparent VE in New York City https://coronavirus.health.ny.gov/covid-19-breakthrough-data

Of course, it is possible that I'm being too optimistic and the vaccinated people in Santa Clara and NYC are being more cautious than their unvaccinated counterparts.

In either case, we should be releasing an update that discounts vaccines based on how long it has been since your last vaccination, so expect VE in microCOVID to decrease.

**Comment by ppanigr** [comment]
Thanks Benjamin for the initial citation. I note that this UK report looks at national registry data from PCR results over a defined time window. The Santa Clara and NYC reports provide summary descriptive stats of positive case rates. What these registry data won't capture is the differences in test-seeking amongst unvaccinated/vaccinated/boosted, and amongst asymptomatic/symptomatic. The UK report also appears to report VE against symptomatic illness (Tables 2,3)

The Danish study mitigates some of this uncertainty by evaluating index and secondary cases within households, so test-seeking behaviors amongst secondary cases are more likely homogenous.

The Ontario study uses a case-negative study design, which removes some of the confounders that can affect interpretation of observational data, more info on this study design here: https://www.nejm.org/doi/full/10.1056/NEJMe2113151

For these reasons, I recommend placing more weight on the results from these 2 studies, although I hear you on the limitations of any data that blends all vaccines without subgroup analyses.

**Comment by beshaya** [comment]
Good point on these registries not capturing differences in test-seeking.
The paper you cited suggests that case-negative studies would
overcompensate for this, though:

> An expected consequence of this design is that case patients and control
patients will enter the study with similar disease manifestations. In a
typical prospective test-negative design, study inclusion is decided before
the test result is obtained,7
<https://www.nejm.org/doi/full/10.1056/NEJMe2113151#> so that selection
bias associated with knowledge of the infection status is avoided. Designs
with retrospective ascertainment of infection status (such as that of the
study by Thompson et al.) are prone to selection bias if, for example,
patients who are highly motivated to be tested and vaccinated also are more
likely to access health care services than those who are not highly
motivated. In this instance, vaccine effectiveness could be underestimated
because vaccinated persons with positive SARS-CoV-2 test results would be
overrepresented.

So the Ontario study has a sample bias towards people who are seeking
healthcare, which biases towards people who are sick.

From that analysis, it is unclear to me whether the UK study biases
underestimate VE (because vaccinated people are more likely to have
resources and trust in healthcare to seek testing) or overestimates VE
(because vaccinated people are more likely to have mild cases or use
at-home testing which is not reported).

If the Danish study had broken out vaccines by type, I agree that the test
strategy is less biased than either of the other studies we're looking at
and I would be all about using it.
